spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/support/GcpPubSubHeaders.java

```diff
@@ -25,7 +25,11 @@
 
 	private static final String PREFIX = "gcp_pubsub_";
 
-	public static final String ACKNOWLEDGEMENT = PREFIX + "pub_sub_acknowledgement";
+	public static final String ACKNOWLEDGEMENT = PREFIX + "acknowledgement";
 
 	public static final String TOPIC = PREFIX + "topic";
+
+	public static final String ACK_ID = PREFIX + "ack_id";
+
+	public static final String SUBSCRIPTION = PREFIX + "subscription_name";
```

Comment:
Our typical pattern is to use underscore for prefix separation, meanwhile the rest of the name is in camel case.
See `KafkaHeaders`, for example: https://github.com/spring-projects/spring-kafka/blob/master/spring-kafka/src/main/java/org/springframework/kafka/support/KafkaHeaders.java